bcda/auth/models_test.go

```diff
@@ -7,10 +7,10 @@ import (
 	"github.com/CMSgov/bcda-app/bcda/auth"
 	"github.com/CMSgov/bcda-app/bcda/database"
 	"github.com/CMSgov/bcda-app/bcda/testUtils"
-	"github.com/jinzhu/gorm"
 	"github.com/pborman/uuid"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/suite"
+	"gorm.io/gorm"
 )
 
 type ModelsTestSuite struct {
```

Comment:
In `TestCMSID` I think we're misusing, the db.Find(), I think that Find + Pluck doesn't really play well. I think we should Model + Where + Pluck